src/Winton.Extensions.Configuration.Consul/ConsulConfigurationClient.cs

```diff
@@ -39,23 +39,37 @@ public IChangeToken Watch(Action<ConsulWatchExceptionContext> onException)
             return _reloadToken;
         }
 
-        private async Task<QueryResult<KVPair>> GetKVPair(QueryOptions queryOptions = null)
+        private async Task<QueryResult<KVPair>> GetKvPair(QueryOptions queryOptions = null)
         {
             using (IConsulClient consulClient = _consulClientFactory.Create())
             {
-                QueryResult<KVPair> result = await consulClient.KV.Get(_source.Key, queryOptions, _source.CancellationToken)
-                    .ConfigureAwait(false);
+                QueryResult<KVPair> result =
+                    await consulClient.KV.Get(_source.Key, queryOptions, _source.CancellationToken).ConfigureAwait(
+                        false);
```

Comment:
If there's to be a new line could the `.ConfigureAwait(` move too? Looks odd with just `false);`.